src/sonic-bgpcfgd/bgpcfgd/managers_as_path.py

```diff
@@ -0,0 +1,50 @@
+from .manager import Manager
+from .log import log_debug, log_warn, log_info
+from swsscommon import swsscommon
+
+T2_GROUP_ASNS = "T2_GROUP_ASNS"
+
+
+class AsPathMgr(Manager):
+    """This class responds to initialize as-path"""
+
+    def __init__(self, common_objs, db, table):
+        """
+        Initialize the object
+        :param common_objs: common object dictionary
+        :param db: name of the db
+        :param table: name of the table in the db
+        """
+        self.directory = common_objs['directory']
+        self.cfg_mgr = common_objs['cfg_mgr']
+        self.constants = common_objs['constants']
+        super(AsPathMgr, self).__init__(
+            common_objs,
+            [],
+            db,
+            table,
+        )
+
+    def set_handler(self, key, data):
+        if key != "localhost":
+            return True
+        asns = None
+        for key_inside, value in data.items():
+            if key_inside == "t2_group_asns":
+                asns = value
+                break
+        log_info("AsPathMgr: Clear asns group {}".format(T2_GROUP_ASNS))
+        self.cfg_mgr.push("no bgp as-path access-list {}".format(T2_GROUP_ASNS))
```

Comment:
What if the asns is partially changed?
Do we want to clear all and reset for each of them?

Reply:
Updated with comparing old config